lib/plugin.coffee

```diff
@@ -24,24 +24,34 @@ cachedScript = (url, options) ->
   $.ajax options
 
 scripts = []
+loadingScripts = {}
 getScript = plugin.getScript = (url, callback = () ->) ->
   # console.log "URL :", url, "\nCallback :", callback
   if url in scripts
     callback()
   else
-    cachedScript url
+    loadScript url
       .done ->
         scripts.push url
         callback()
-      .fail ->
+      .fail (_jqXHR, _textStatus, err) ->
+        console.log('Failed to load plugin:', url, err)
```

Comment:
Although getScript() is part of the plugin machinery, it is also exposed as wiki.getScript and used elsewhere to load supporting scripts. Might be better to use something like `console.log('getScript: Failed to load:', url, err)`

Reply:
Good catch. I'll just make this change to keep this pr moving.